include/seqan3/io/alignment_file/detail.hpp

```diff
@@ -43,13 +43,15 @@
 #include <sstream>
 
 #include <seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp>
+#include <seqan3/core/concept/tuple.hpp>
 #include <seqan3/std/concepts>
 #include <seqan3/std/ranges>
 
 #include <range/v3/view/zip.hpp>
 
 namespace seqan3::detail
 {
+//!\privatesection
```

Comment:
I don't think this makes a difference? How can non-members be private?

Reply:
You need to discuss this with doxygen. it solves the issue for me

Reply:
touché :rofl: